spec/models/organization_spec.rb

```diff
@@ -106,6 +106,12 @@
       expect(organization.errors[:slug].to_s.include?("taken")).to be true
     end
 
+    it "takes used partial into account" do
```

Comment:
```suggestion
    it "takes sitemap into account" do
```
I _think_ this is what the test is actually testing?